### PR TITLE
Sil split

### PR DESCRIPTION
**Pull request by madhas8**
- [x] I have read the [CONTRIBUTION GUIDE][contributing]

- [ ] This is a bug

- [x] This is a feature request

- [ ] This is a syntax/style fix

- [ ] This is an improvement to code quality

---
**Please describe your changes below:**
(Note: A clear description of changes is required to approve any pull requests)

_"Changed the implementation for verification because..."_

[contributing]: http://dellemc-symphony.readthedocs.io/en/latest/contributingtosymphony.html

### CODE REVIEW

File: src/main/java/com/dell/cpsd/hdp/capability/registry/dto/RequestQueue.java [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    {
+        return autoDelete;
+    }
+
+    public void setAutoDelete(boolean autoDelete)
+    {
+        this.autoDelete = autoDelete;
+    }
+
+    public java.util.Map<java.lang.String, java.lang.Object> getArguments()
```
Comment (ganapn) [inline src/main/java/com/dell/cpsd/hdp/capability/registry/dto/RequestQueue.java]: Looks like fully qualified types are used by mistake. Please change all to simple types and import them

File: src/main/java/com/dell/cpsd/hdp/capability/registry/exceptions/CapabilityRegistrationRuntimeException.java [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+package com.dell.cpsd.hdp.capability.registry.exceptions;
+
+/**
+ * CapabilityRegistrationRuntimeException - Custom runtime exception class for capability registration
+ * <p>
+ * Copyright &copy; 2017 Dell Inc. or its subsidiaries. All Rights Reserved. Dell EMC Confidential/Proprietary Information
+ * </p>
+ */
+public class CapabilityRegistrationRuntimeException extends RuntimeException
```
Comment (ganapn) [inline src/main/java/com/dell/cpsd/hdp/capability/registry/exceptions/CapabilityRegistrationRuntimeException.java]: Why is this a RuntimeException? Why should this be not checked?

Comment (madhas8) [reply src/main/java/com/dell/cpsd/hdp/capability/registry/exceptions/CapabilityRegistrationRuntimeException.java]: This exception is thrown at start up of the app when registration fails. Failure of registration can be determined only at runtime.

**Comment by ganapn** [comment]
Resolve conflicts.

**Pull request closed without merging**